css/styles.css

```diff
@@ -709,3 +710,458 @@ a {
   color: #172b4d;
   transform: 0.8s;
 }
+
+.nav-link .navigation-bar {
+  display: none;
+}
+
+/** Desktop Media css styles **/
+@media only screen and (min-width: 768px) {

```

Comment:
- [ ] At desktop width , there is horizontal scroll present in your project, it should not be present. Let's fix this issue. Cheers👍 

![image](https://user-images.githubusercontent.com/60618877/179269680-5dfb8e6d-c12c-429b-97fc-965d357c4ce1.png)